lib/models/BucketInfo.ts

```diff
@@ -240,6 +241,10 @@ export default class BucketInfo {
             READ_ACP: [],
         };
 
+        if (tags) {
+            assert(Array.isArray(tags));
```

Comment:
We should probably also check what is in each tag

Reply:
I found that the validation function internally used was not using the exact same format so I created another one that is exported and that will throw an error instead of returning one. It's the same function but using a different format, i.e., the format that we are supposed to use when we create a `BucketInfo` object. The other function deals with XML.

https://github.com/scality/Arsenal/pull/1933/commits/08703718b843b40a802adb9aaafdf7840cd05baa#diff-7aa7eeaed88643ea97d57a5f15ddc2e64c380190641c53bb0c3f64d67807a475R120

So now I am calling this validation function in the constructor, after assigning the default value (`[]`) if the parameter is undefined